packages/form-layout/src/vaadin-form-layout-styles.js

```diff
@@ -91,10 +92,9 @@ export const formLayoutStyles = css`
   }
 
   :host([auto-responsive]) #layout ::slotted(*) {
+    grid-column: 1 / span min(var(--_colspan, 1), var(--_rendered-column-count));
```

Comment:
Let's move this line under the CSS variable declarations.

Reply:
Done.